understanding/non-text-content.html

```diff
@@ -627,13 +472,7 @@ <h4>Situation A: If a short description can serve the same purpose and present t
                         
                         <li>
                            
-                           <a class="general" href="https://w3c.github.io/wcag/techniques/general/G94">G94: Providing short text alternative for non-text content that serves the same purpose
-                              and presents the same information as the non-text content</a> using one of the following 
-                           <strong>
-                              
-                              <a href="#text-equiv-all-situation-a-shorttext">Short text alternative techniques for Situation A</a>
-                              
-                              </strong>:
+                           次に挙げる<strong><a href="#text-equiv-all-situation-a-shorttext">状況 A における短いテキストによる代替の達成方法</a></strong>のいずれかを用いて、<a href="#text-equiv-all-situation-a-shorttext">G94: 非テキストコンテンツに対して、それと同じ目的を果たし、かつ同じ情報を示す、短いテキストによる代替を提供する</a>:
```

Comment:
今のところ、Techniques for WCAG 2.2（仮称：WCAG 2.2テクニック集）は未翻訳ですので、リンクテキストも翻訳せずにしてもらえれば。個々に限っていえばリンクURLも置き換えがうまくいっていないようですので、お手数ですが、あわせて原文に書き戻してもらえればと。
```suggestion
                           次に挙げる<strong><a href="#text-equiv-all-situation-a-shorttext">状況 A における短いテキストによる代替の達成方法</a></strong>のいずれかを用いて、<a class="general" href="https://w3c.github.io/wcag/techniques/general/G94">G94: Providing short text alternative for non-text content that serves the same purpose and presents the same information as the non-text content</a>:
```